Source/Core/DolphinQt2/NetPlay/NetPlayDialog.cpp

```diff
@@ -446,6 +449,12 @@ void NetPlayDialog::UpdateGUI()
     m_hostcode_action_button->setText(tr("Copy"));
     m_hostcode_action_button->setEnabled(true);
   }
+
+  if (m_old_player_count != player_count)
+  {
+    Discord::UpdateDiscordPresence(player_count, m_hostcode_label->text().toUtf8());
```

Comment:
This is not a valid way to get the host code or IP address. The value of this won't be the host code of the "Room ID" topn is not selected, and the IP address is only the interface adress, not the user's external IP. You'll need to query the IP from a remote server. Ask one of the project managers about this one, as we'd rather not rely on a service out of our control for this.